incubator/kube-backup/values.yaml

```diff
@@ -0,0 +1,39 @@
+# Default values for lamp-backup.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+define:
+  - &MYSQL_SCHEDULE "0 0 * * *"   # Mysql backup schedule
+  - &MYSQL_DATABASE ""            # Mysql Database name
+  - &NAMESPACE ""                 # Namespace of mysql pod
+  - &SELECTOR ""                  # Selector to find mysql pod
+  - &CONTAINER ""                 # Mysql container name in pod
+  - &S3BUCKET ""                  # S3 bucket name ( destination )
+  - &IAM_ROLE ""                  # IAM Role to assume for s3 bucket access
+
+image:
+  repository: cloudposse/kube-backup
+  tag: latest
+  pullPolicy: Always
+resources:
+  limits:
+    cpu: 100m
+    memory: 128Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
+
+mysql:
+  enabled: true
+  schedule: *MYSQL_SCHEDULE
+  concurrencyPolicy: Allow
```

Comment:
This seems like a bad idea. Would we want to back up the *same* database concurrently if the job is taking a long time? That would lead to cascading failures.